### ISSUE TITLE
[10/4] FFYR homepage card updates

### ISSUE THREAD

**Issue by djst** [mozilla/bedrock#6196]
This work supports the Fight for Your Rights campaign

Success Criteria:

    New cards live on 10/4 (content delivered from FMB/MJ)

Currently blocked on specific cards from MJ

**Comment by alexgibson** [comment]
@djst is this a duplicate of https://github.com/mozilla/bedrock/issues/6154 ?

**Comment by ejregithub** [comment]
@alexgibson @djst - yes, this is a dup to 6154, closing this one
